Prose/ProseLib/Sources/ProseCoreTCA/Types/JID+ProseCore.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+@_implementationOnly import ProseCoreClientFFI
+import SharedModels
+
+extension JID {
+    init(bareJid: ProseCoreClientFFI.BareJid) {
+        self.init(rawValue: "\(bareJid.node ?? "")@\(bareJid.domain)")
```

Comment:
Why do you use `.init(rawValue:)` here? Couldn't you use the `.init(node:domain:resource:)` initializer?

Reply:
Because it wasn't public and not suggested by Xcode. I've changed that.